CareKitStore/CareKitStoreTests/Streaming/TestCoreDataQueryMonitor.swift

```diff
@@ -214,6 +214,7 @@ class TestCoreDataQueryMonitor: XCTestCase {
         XCTAssertEqual(expectedTitles, observedTitles)
     }
 
+    #if compiler(>=5.8.0)
```

Comment:
@gavirawson-apple I added this here because GitHub actions currently only supports up to macOS 12 and Xcode 14.2. We can add a note to remove this when actions releases a macOS 13 image. More info here: https://github.com/actions/runner-images/issues/6426#issuecomment-1479974889

My observations over the last couple of years, it takes actions awhile to upgrade to the latest macOS, I suspect this will be June/July